config/rbac/role.yaml

```diff
@@ -34,6 +34,23 @@ rules:
   - get
   - list
   - watch
+- apiGroups:
```

Comment:
does this work for non-"unmanaged" installations?
@matancarmeli7 which PR should enforce it? https://github.com/IBM/ibm-block-csi-operator/pull/220? could we ask to prioritize it?

Reply:
how would it work for any other object?

Reply:
I'm referring to all changes in this PR for this file

Reply:
@oriyarde #226 this is the one, and yes I will try along with the GitHub actions one

Reply:
thanks

Reply:
olm files updated